airflow/operators/cassandra_to_gcs.py

```diff
@@ -209,29 +201,38 @@ def _write_local_schema_file(self, cursor):
         schema = []
         tmp_schema_file_handle = NamedTemporaryFile(delete=True)
 
-        for name, type in zip(cursor.column_names, cursor.column_types):
-            schema.append(self.generate_schema_dict(name, type))
+        for name, type_ in zip(cursor.column_names, cursor.column_types):
+            schema.append(self.generate_schema_dict(name, type_))
         json_serialized_schema = json.dumps(schema).encode('utf-8')
 
         tmp_schema_file_handle.write(json_serialized_schema)
         return {self.schema_filename: tmp_schema_file_handle}
 
-    def _upload_to_gcs(self, files_to_upload):
+    def _upload_to_gcs(self, files_to_upload: Dict):
```

Comment:
```suggestion
    def _upload_to_gcs(self, files_to_upload: Dict[str, str]):
```

Reply:
Are you sure about that?

Reply:
Not sure. But MyPy should be able to check it if we specify it here.